CMakeLists.txt

```diff
@@ -635,93 +620,112 @@ elseif(${PLATFORM_NAME} STREQUAL Mac OR ${PLATFORM_NAME} STREQUAL Linux)
+                DESTINATION "${OPENSIM_INSTALL_SIMBODYDIR}"
+                FILES_MATCHING PATTERN "*SimTK*.dll")
+        install(DIRECTORY "${Simbody_ROOT_DIR}/"
+                DESTINATION "${OPENSIM_INSTALL_SIMBODYDIR}"
+                # There's no need to install Simbody examples or docs.
+                PATTERN "*examples/*" EXCLUDE
+                PATTERN "*doc/*" EXCLUDE
+                # Don't copy lapack, blas, etc. into sdk/Simbody/bin/; they are
+                # already copied into bin/.
+                PATTERN "*.dll" EXCLUDE
+                PATTERN "*.exe" EXCLUDE)
+        # DLLs are handled by OpenSimCopyDependencyDLLsForWin(), but we still
+        # need to copy simbody-visualizer.exe.
+        install(DIRECTORY "${Simbody_BIN_DIR}/"
+                DESTINATION "${CMAKE_INSTALL_BINDIR}"
+                FILES_MATCHING PATTERN "*.exe")
+        # TODO We are copying Simbody DLLs to both the bin and sdk/Simbody/bin
+        # folders; we would prefer to only copy to the former, but
+        # SimbodyTargets-*.cmake expect dlls in sdk/Simbody/bin. In the future,
+        # we can edit the *.cmake files to no longer expect these DLLs.
     else()
-        # TODO simbody-visualizer executable probably isn't even there.
-        file(GLOB SIMBODY_BIN_FILES
-            ${Simbody_BIN_DIR}/simbody*) # executables only (e.g., visualizer).
-        # If the LIB_DIR is some common place for libraries (e.g., /usr/lib/),
-        # we only want to copy over libraries that have SimTK in their name.
-        file(GLOB SIMBODY_LIB_FILES
-            ${Simbody_LIB_DIR}/*SimTK*.so
-            ${Simbody_LIB_DIR}/*SimTK*.so.*
-            ${Simbody_LIB_DIR}/*SimTK*.dylib)
-        file(GLOB SIMBODY_ARCHIVE_FILES
-            ${Simbody_LIB_DIR}/*SimTK*.a)
+        # We would have liked to use install(DIRECTORY) on UNIX, but the
+        # dependencies may be installed in system-wide locations and we don't
+        # want to copy extraneous files/directories; install(DIRECTORY) creates
+        # a directory in DESTINATION for *every* directory in DIRECTORY, even
+        # if we do not want to copy any files from some of the directories in
+        # DIRECTORY.
+
+        # include
+        file(RELATIVE_PATH install_simbodyincludedir
+            "${Simbody_ROOT_DIR}" "${Simbody_INCLUDE_DIR}")
+        install(DIRECTORY "${Simbody_INCLUDE_DIR}/"
+                DESTINATION "${OPENSIM_INSTALL_SIMBODYDIR}/${install_simbodyincludedir}")
+    
+        # lib (and cmake, pkgconfig)
+        file(RELATIVE_PATH install_simbodylibdir 
+            "${Simbody_ROOT_DIR}/" "${Simbody_LIB_DIR}/")
+        file(GLOB_RECURSE simbody_desired_lib_files
+            RELATIVE "${Simbody_LIB_DIR}"
+            "${Simbody_LIB_DIR}/*SimTK*" "${Simbody_LIB_DIR}/*Simbody*.cmake"
+            "${Simbody_LIB_DIR}/*SampleCMakeLists.txt*"
+            "${Simbody_LIB_DIR}/*simbody.pc")
+        foreach(simbody_lib_file ${simbody_desired_lib_files})
```

Comment:
simbody.pc is a pkg-config file, mostly for use on UNIX (alternative to using SimbodyConfig.cmake). I just looked at the [contents of this file](https://github.com/simbody/simbody/blob/master/cmake/pkgconfig/simbody.pc.in), and I just realized that it does not work if the simbody installation has been relocated. So I will remove the copying of this file into the OpenSim installation.

We expect UNIX users to have their own lapack, blas, pthread, etc. On UNIX, we only need the 3 simbody libraries.

I think my tests were sufficiently thorough (I changed the name of the install directory to assess portability). However, I did not test using the installation on other machines. 

I'm happy to address any issues that arise.